draft-scurtescu-secevent-event-stream-management-api.md

```diff
@@ -449,9 +449,9 @@ the Event Receiver as a result of the above request:
   "iat": "1493856000",
   "events": [
     "urn:ietf:params:secevent:event-type:core:verification" : {
-      "state": "VGhpcyBpcyBhbiBleGFtcGxlIHN0YXRlIHZhbHVlLgo=",
```

Comment:
If you don't mind then let's use the trailing commas.

It is not only the visual aspect, but also the line count for the diff is off without them.

We are not targeting any browsers and this is only sample data anyhow.